front/app/containers/ProjectsShowPage/shared/events/scrollToEventState.ts

```diff
@@ -0,0 +1,6 @@
+let scrollToEventId: string | null = null;
```

Comment:
I know this looks a bit hacky, but it was easier than using the `eventEmitter`. The problem is that when the `/:locale/projects` route is loaded, you ideally want to have the `eventId` available already. This way you can be sure that it is available. With the event emitter, you are never sure when exactly the data arrives. In practice, however, since you are also waiting for data fetching in `ProjectsShowPage`, you will have it before the data is fetched there. But still, this solution is simple and uses less code than the `eventEmitter` one. If this is a terrible idea I can change it of course.

Reply:
Looks hacky indeed. I don't know why though, so if it works, I don't have any good incentive of having you change it, though, except the case I've made for the url. As for the event solution, I believe there should be a solution to that, the events live in streams, so you should be able to push an event to the stream and catch it when you're ready, you don't have to have the listener up before you send the event.